### ISSUE TITLE
MS_EXPOSE_REP cannot detect nested class

### ISSUE THREAD

**Issue by Belle-PL** [spotbugs/spotbugs#2042]
Hi, I found a false negative about the rule [MS_EXPOSE_REP](). Please read the following case, PMD should have reported a warning in line 5,but it doesn't. I think this is caused by the nest class. Thanks for your consideration.

```java
public class C1 {
  private static String[] s2;
  public class C2 {
    public String[] getS2() {
      return s2;  // should report a warning here
    }
  }
}
```
Version: Latest

**Comment by ThrawnCA** [comment]
> PMD should have reported

I think you mean SpotBugs?

**Comment by Belle-PL** [comment]
@ThrawnCA Hi, sorry for the mistake. I have fixed it 😸

**Comment by baloghadamsoftware** [comment]
Thank you for reporting this. I am just checking what I can do here.
